muckrock/core/context_processors.py

```diff
@@ -6,9 +6,16 @@
 from django.conf import settings as django_settings
 from django.utils.functional import SimpleLazyObject
 
+# Standard Library
+import hashlib
+
 # Third Party
 from constance import config
 
+# MuckRock
+# Local
```

Comment:
delete this `# Local `, sometimes the import lib won't clear doubled up comments automatically